### PR TITLE
Add zipkin tracer initializer

### PR DESCRIPTION
**Pull request by klingerf**
Adding a new trace initializer for configuring Finagle's zipkin-tracer via a linkerd config file. Let me know if this is the appropriate place to put this code within the project or if it should go elsewhere.

### CODE REVIEW

**Comment by esbie** [comment]
lgtm

File: linkerd/tracer/zipkin/src/main/scala/io/l5d/zipkin.scala [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  port: Option[Int],
+  sampleRate: Option[Double]
+) extends TracerConfig {
+
+  @JsonIgnore
+  override def newTracer(): Tracer = new Tracer {
+
+    private[this] def getHost: String = host.getOrElse("localhost")
+    private[this] def getPort: Int = port.getOrElse(9410)
+    private[this] def getSampleRate: Float =
```
Comment (olix0r) [inline linkerd/tracer/zipkin/src/main/scala/io/l5d/zipkin.scala]: tioli: i don't think it adds much value to have `def getFoo` given that these just get called once, immediately.  Could even inline them into ZipkinTrace.mk...

nbd one way or the other

**Comment by olix0r** [comment]
:star: lgtm, one tioli

**Comment by adleong** [comment]
:star: :memo: :penguin:

File: linkerd/tracer/zipkin/src/main/scala/io/l5d/zipkin.scala (hunk 1 above)
Comment (klingerf) [inline linkerd/tracer/zipkin/src/main/scala/io/l5d/zipkin.scala]: Good suggestion, have updated and inlined them.

**Pull request merged**